testbed/testbed/utils/graphsync.go

```diff
@@ -109,9 +111,13 @@ func (n *GraphsyncNode) Fetch(ctx context.Context, c cid.Cid, peers []PeerInfo)
 		return nil, errors.New("no suitable seed found")
 	}
 	p := peers[seedIndex].Addr.ID
+
+	start := time.Now()
 	resps, errs := n.gs.Request(ctx, p, cidlink.Link{Cid: c}, selectAll)
 	for range resps {
 	}
+	fmt.Println("TIME SINCE START: ", time.Since(start))
```

Comment:
@hannahhoward @adlrocha 

I think this is the correct time it took for the GS request, right ? I see a difference of almost 20 seconds for a 1G file if we don't measure it here and measure it after the call returns.

Reasons:

1. The `n.gs.Request` call already writes the blocks to the datastore.
2. We do one more read in the `n.dserv.Get(ctx, c)` call below (I think ?)
3. In the transfer code, we call `err = files.WriteTo(rcvFile, "/tmp/"+strconv.Itoa(t.tpindex)+time.Now().String())` and do one more write of the whole file. 

So, we should ONLY measure 1. Wdyt ?

Reply:
@hannahhoward is definitely the expert here. I recall that in Bitswap we did like that because if you don't explicitly write the file, UnixFS doesn't know if there are blocks pending, and Bitswap doesn't traverse the full DAG. I don't think this is the case for GS, but can't be sure.

Reply:
ha! I know what this is about and it indicates why you make be getting slow results for Graphsync transfer as well. There is a 5 millisecond delay on blockstore reads by default! For a 1GB file, at default chunking settings for this test (256K), that's >4000 blocks x 5 milliseconds = 20 seconds!

You are going to want to set bstore_delay_ms = 0 in your composition for a fair comparison with any of your other protocols.

Reply:
I've added support for using a Badger data store in addition to the mutex wrapped in-memory store with an artificial delay. Will get back with results using:

1. Badger
2. BlockStore Delay of 0.
3. BlockStore Delay of 5ms.